migrations/versions/5b7164faeda5_sampledballotdraw_contest_id.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=invalid-name
+"""SampledBallotDraw.contest_id
+
+Revision ID: 5b7164faeda5
+Revises: fa342e75506b
+Create Date: 2020-07-07 18:02:22.237055+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "5b7164faeda5"
+down_revision = "fa342e75506b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add SampledBallotDraw.contest_id
+    op.add_column("sampled_ballot_draw", sa.Column("contest_id", sa.String(length=200)))
+    op.create_foreign_key(
+        op.f("sampled_ballot_draw_contest_id_fkey"),
+        "sampled_ballot_draw",
+        "contest",
+        ["contest_id"],
+        ["id"],
+        ondelete="cascade",
+    )
+
+    # Fill in the targeted contest id for existing SampledBallotDraws
+    op.execute(
+        """
+        UPDATE sampled_ballot_draw
+        SET contest_id = contest.id
+        FROM contest
+        JOIN election ON election.id = contest.election_id
+        JOIN round ON election.id = round.election_id
+        WHERE round.id = sampled_ballot_draw.round_id
+        AND contest.is_targeted
+        """
```

Comment:
this works even if there's more than one targeted contest?

Reply:
at this point in time, we know there's only one targeted contest per election in the prod db

Reply:
but is that happenstance? Could the data model have more than one? So maybe `limit 1` just to be safe?